recipe/meta.yaml

```diff
@@ -35,10 +35,11 @@ test:
 
 about:
   home: https://github.com/tqdm/tqdm
-  license: Mozilla Public License 2.0 (MPL 2.0) or MIT License
+  license: MPLv2.0, MIT
   summary: 'A Fast, Extensible Progress Meter'
 
 extra:
   recipe-maintainers:
+    - casperdcl
```

Comment:
@pmlandwehr @ocefpaf no problem, as @lrq3000 said it just seems more future proof (and probably more morally and legally correct) to support adding the organisation, but we're happy with individual access, thanks.